doc/dev_manual/image_viewer.rst

```diff
@@ -0,0 +1,501 @@
+.. _ch-image-viewer-operations:
+
++++++++++++++++++++++++++++++
+Ginga Image Viewer Operations
++++++++++++++++++++++++++++++
+
+This chapter describes the operations supported by the basic ginga image
+viewer and how to access them programatically.  For all of these
+API examples we assume the viewer object is contained in variable `v`.
```

Comment:
Use double-backticks. Single ones will attempt to x-ref (link) them.

Also, would be helpful to add a code snippet to see how `v` is set. Example:
```python
from ginga import ???

v = ???
```